dist/index.html

```diff
@@ -28,8 +27,7 @@
         <div class="row top-buttons">
             <div class="col-md-12">
                 <div class="button-container">
-                    <a href="https://hackersatupc.typeform.com/to/Bt9YQF">Apply</a>
-                    <a href="mailto:sponsor@hackupc.com">Sponsor</a>
+                    <h2>Applications are closed. See you all at HackUPC</h2>
```

Comment:
THis will go in PR #35 as it needs to be finished today. I will use it from here.